ui/src/components/Dashboard/SidebarList.jsx

```diff
@@ -66,14 +66,14 @@ export default class SidebarList extends Component {
 
         <SidebarListItem to="/worker-manager" icon={<HexagonSlice4 />}>
           Worker Manager
-        </SidebarListItem>
+        </SidebarListItem> 
 
         <SidebarListItem to="/provisioners" icon={<MixcloudIcon />}>
           Provisioners
         </SidebarListItem>
 
         <SidebarListItem to="/purge-caches" icon={<BackupRestoreIcon />}>
-          Purge Cache
+          Purge Caches 
```

Comment:
@lailahgrant You just need to change this line, All other changes are irrelevant. That's all you have to do in this PR 😄

Reply:
@vishakhanihore thanks very much....  you know actually I did  that .

Reply:
![i did](https://user-images.githubusercontent.com/28113644/68969434-04142600-07f6-11ea-94e1-f94ec570a1cb.PNG)

Reply:
@helfi92 , please help me review this, thanks :sweat: